### PR TITLE
Migrate Tabs component from kaizen-legacy

### PR DESCRIPTION
**Pull request by dougmacknz**
Notes:
* Added classNameOverride and native props to each of the components so that I could replace Box usages out in the stories.
* Got rid of the `keyboardActivation` prop because I couldn't think of a good reason to need it, and no one has used it.

### CODE REVIEW

File: packages/components/src/Tabs/_docs/Tabs.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import { Meta, StoryObj } from "@storybook/react"
+import { Button } from "~components/Button"
+import { Card } from "~components/Card"
+import { Text } from "~components/Text"
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "../index"
+
+const meta = {
+  title: "Components/Tabs",
+  component: Tabs,
+  args: {},
```
Comment (gyfchong) [inline packages/components/src/Tabs/_docs/Tabs.stories.tsx:11]: You can add the `children` here which should apply them to all stories

**Review by gyfchong — DISMISSED** [review]
Just a nit to make your stories neater

File: packages/components/src/Tabs/_docs/Tabs.stories.tsx (hunk 1 above)
Comment (dougmacknz) [reply packages/components/src/Tabs/_docs/Tabs.stories.tsx]: Nice good call

File: packages/components/src/Tabs/_docs/Tabs.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    ),
+  },
+} satisfies Meta<typeof Tabs>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
```
Comment (gyfchong) [inline packages/components/src/Tabs/_docs/Tabs.stories.tsx:41]: Add `chromatic: { disable: false }` here

**Review by gyfchong — APPROVED** [review]

**Pull request merged**